project/src/main/java/com/google/sps/servlets/EditUserServlet.java

```diff
@@ -78,6 +82,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         }else{
```

Comment:
Add spaces between if else and curly braces

Reply:
done for all files.